packages/components/nodes/llms/Fireworks/Fireworks.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    getBaseClasses,
+    getCredentialData,
+    getCredentialParam,
+    ICommonObject,
+    INode,
+    INodeData,
+    INodeOptionsValue,
+    INodeParams
+} from '../../../src'
+import { Fireworks } from '@langchain/community/llms/fireworks'
+import { getModels, MODEL_TYPE } from '../../../src/modelLoader'
+import { BaseCache } from '@langchain/core/caches'
+
+class Fireworks_LLMs implements INode {
+    label: string
+    name: string
+    version: number
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    credential: INodeParams
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Fireworks'
+        this.name = 'fireworks'
+        this.version = 1.0
+        this.type = 'Fireworks'
+        this.icon = 'fireworks.png'
+        this.category = 'LLMs'
+        this.description = 'Wrapper around Fireworks API for large language models'
+        this.baseClasses = [this.type, ...getBaseClasses(Fireworks)]
+        this.credential = {
+            label: 'Connect Credential',
+            name: 'credential',
+            type: 'credential',
+            credentialNames: ['fireworksApi']
+        }
+        this.inputs = [
+            {
+                label: 'Cache',
+                name: 'cache',
+                type: 'BaseCache',
+                optional: true
```

Comment:
We need to have the model field parameter, example - https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/chatmodels/ChatHuggingFace/ChatHuggingFace.ts#L41

Since there are so many models, we can just make it as a type: 'string field, and get rid of the listModels method

Reply:
done